components/files.mdx

```diff
@@ -10,6 +10,65 @@ This works for any file type you want to make available to your users, including
 
 Files must be less than 20 MB for images and 100 MB for other file types.
 
+## Supported file extensions
+
+Mintlify supports the following static file extensions.
+
+### Image formats
+- `.jpeg`, `.jpg`, `.jfif`, `.pjpeg`, `.pjp`
+- `.png`
+- `.svg`, `.svgz`
+- `.ico`
+- `.webp`
+- `.gif`, `.apng`
+- `.avif`
+- `.bmp`
+- `.tiff`, `.tif`
+
+### Video formats
+- `.mp4`
+- `.webm`
+- `.ogg`
+- `.avi`
+- `.mov`
+- `.wmv`
+- `.flv`
+- `.mkv`
+- `.m4v`
+
+### Audio formats
+- `.mp3`
+- `.wav`
+- `.flac`
+- `.aac`
+- `.ogg`
+- `.wma`
+- `.m4a`
+
+### Document formats
+- `.pdf`
+- `.txt`
+- `.doc`, `.docx`
+- `.xls`, `.xlsx`
+- `.ppt`, `.pptx`
+- `.rtf`
+- `.odt`, `.ods`, `.odp`
+
+### Data formats
+- `.csv`
+- `.tsv`
+- `.xml`
+
+### Font formats
+- `.woff`, `.woff2`
+- `.ttf`
+- `.eot`
+- `.otf`
```

Comment:
```suggestion
```

I'm going to remove this section because it could cause confusion with file types supported for custom fonts and it's an edge case that someone wants to host a bunch of different font files for users to download.